cmd/dkvsrv/main.go

```diff
@@ -93,9 +113,28 @@ func main() {
 	srvrRole := toDKVSrvrRole(dbRole)
 	srvrRole.printFlags()
 
+	// Create the region info which is passed to DKVServer
+	regionInfo := &serverpb.RegionInfo{
+		DcID:            dcID,
+		NodeAddress:     dbListenAddr,
+		Database:        database,
+		VBucket:         vBucket,
+		Status:          serverpb.RegionStatus_INACTIVE,
+		MasterHost:      nil,
+		NexusClusterUrl: nil,
+	}
+
+	statusPropagator, clusterInfoGetter := getDiscoveryClients()
+
+	if (statusPropagator != nil) {
```

Comment:
This null check is currently required because

Discovery server won't have a discovery client as client needs a server to be started. and have made discovery client mandatory for server start. It can be made retryable and thus non mandatory. will do that in the next PR When supporting multiple regions in a node and remove the null check